src/docs/user.yaml

```diff
@@ -0,0 +1,26 @@
+paths:
+  /api/user/{id}:
+    get:
+      tags:
+        - get users
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          required: true
+          description: 'ID of the user'
+          examples:
+            'real ID of a deleted user':
```

Comment:
why are you giving id of the deleted user?

Reply:
It was the only id I could put hands on. Now new added.